Kudu.Core/Deployment/DeploymentHelper.cs

```diff
@@ -49,5 +51,46 @@ public static bool IsDefaultWebRootContent(string webroot)
 
             return false;
         }
+
+        public static void PurgeZipsIfNecessary(string sitePackagesPath, ITracer tracer)
+        {
+            int DEFAULT_PURGE_VAL = 5;
+            int MIN_PURGE_VAL = 1;
+
+            var totalAllowedZips = DEFAULT_PURGE_VAL;
+            if (System.Environment.GetEnvironmentVariable("PURGE_VAL") != null)
```

Comment:
- Use SCM_ naming pattern for consistency with other Kudu settings
- Define a constant in `SettingsKeys.cs`
- Use `IDeploymentSettingsManager` to get to the value

e.g. search for `SettingsKeys.DisableSubmodules` for an example that uses this pattern.

Reply:
I made the update, but please let me know if there's a better place to keep the constants for default and min values `DEFAULT_ALLOWED_ZIPS` and `MIN_ALLOWED_ZIPS`. Thanks again for the feedback.